frontend/packages/dev-console/integration-tests/support/step-definitions/common/operators.ts

```diff
@@ -1,24 +1,10 @@
 import { Given } from 'cypress-cucumber-preprocessor/steps';
-import { operatorsPO } from '@console/dev-console/integration-tests/support/pageObjects';
-import { operators, switchPerspective } from '../../constants';
-import { perspective, operatorsPage, installOperator, verifyAndInstallOperator } from '../../pages';
+import { operators } from '../../constants';
+import { verifyAndInstallOperator } from '../../pages';
+import { verifyAndInstallWebTerminalOperator } from '../../pages/functions/installOperatorOnCluster';
 
 Given('user has installed Web Terminal operator', () => {
-  perspective.switchTo(switchPerspective.Administrator);
-  operatorsPage.navigateToInstallOperatorsPage();
-  cy.get(operatorsPO.installOperators.search)
-    .should('be.visible')
-    .clear()
-    .type(operators.WebTerminalOperator);
-  cy.get('body', {
-    timeout: 50000,
-  }).then(($ele) => {
-    if ($ele.find(operatorsPO.installOperators.noOperatorsFound)) {
-      installOperator(operators.WebTerminalOperator);
-    } else {
-      cy.log('Serverless operator is installed in cluster');
-    }
-  });
+  verifyAndInstallWebTerminalOperator();
```

Comment:
why not use `verifyAndInstallOperator(operators.WebTerminalOperator)` instead? it should already take care of majority of the steps